lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -312,4 +313,33 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
             return bucket.count >= mincount;
         };
     }
+
+    private getCorrespondingFilterQuery (field: FacetField, bucketLabel: string): string {
```

Comment:
field doesn't' look the right name do you see in the code after you find yourself use field. field. you can call it facetField.